cmd/gorepomod/internal/repo/dotgitdata.go

```diff
@@ -136,3 +169,33 @@ func (dg *DotGitData) checkModules(modules []*protoModule) error {
 	}
 	return nil
 }
+
+func (dg *DotGitData) checkModulesLocal(modules []*protoModule) error {
+	for _, pm := range modules {
+		file := filepath.Join(pm.PathToGoMod(), goModFile)
+		_, err := os.Stat(file)
+		if err != nil {
+			return fmt.Errorf(
+				"cannot find go.mod file in %q", file)
+		}
+		pm.ShortNameWithLocalFlag(dg.RepoPath())
+	}
+	return nil
+}
+
+// Extract local prefix from origin url information retrieved from .git
+func getLocalPrefix(dgAbsPath string) string {
+	_, err := os.Stat(dgAbsPath + ".git")
+	if err != nil {
+		fmt.Errorf(".git directory does not exist int path %q", dgAbsPath)
+	}
+
+	out, err := exec.Command("git", "config", "--get", "remote.origin.url").Output()
+	if err != nil {
+		fmt.Errorf("failed extracting git information: %q", err.Error())
+	}
+
+	var originUrlString string = string(out)
+	var localPrefix string = utils.ParseGitRepositoryPath(originUrlString)
+	return localPrefix
```

Comment:
Do you have any special reason to use a few temporary variables?
I think we can write one line from the above 3 lines, and I suggest using any comment line if you need to explain what is happening.

Reply:
None in particular, I forgot to refactor it after debugging